docs/getting-started.md

```diff
@@ -44,4 +44,72 @@ Een lijst met andere ondersteunde generator opties kun je vinden in de [Generato
+
+In Postman kun je de 'Bevraging Persoon' OpenAPI specificatie importeren en visueel de verschillende endpoints aanroepen. Volg deze stappen om het OpenAPI specificatie bestand te importeren:
+
+1.Klik op de Import button om de Import dialog box te openen
+
+2.Selecteer 'Import From Link' tab, plak de volgende url in de 'Enter a URL and press Import' textbox en klik op de Import button
+
+``` url
+https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-BRP-Update-API/master/specificatie/genereervariant/openapi.yaml
+```
+
+3.Klik op de Next button om een Postman collectie te genereren uit het OpenAPI specificatie bestand
+
+### Configureer de url en api key
+
+1. Klik bij "BRP Update API" op de drie bolletjes.
+2. Klik vervolgens op Edit
+3. Selecteer tabblad "Authorization"
+4. Kies TYPE "API Key"
+5. Vul in Key: "x-api-key", Value: de API key die je van Cathy hebt ontvangen, Add to: "Header"
+6. Selecteer tabblad "Variables"
+7. Vul bij baseUrl INITIAL VALUE en bij CURRENT VALUE: `https://www.haalcentraal.nl/haalcentraal/api/brpupdate`
+8. Klik op de knop Update
+
+### Gebruik van de BRP-update API testvoorziening
+
+Op de BRP-update API in de Haal Centraal demo-omgeving worden elke dag enkele personen (burgerservicenummers) als wijziging toegevoegd. Op deze manier kan je voor elke dag nieuwe wijzigingen ontvangen.
+
+Om wijzigingen te ontvangen moet je eerst een volgindicatie toevoegen met PUT /volgindicaties/:burgerservicenummer. Daarbij vul je in Postman bij Path Variable 'burgerservicenummer' als VALUE het burgerservicenummer in van de persoon waarop je wijzigingen wilt ontvangen. 
+
+Bij het zetten van een volgindicatie moet ook een request body worden opgenomen, waarmee een einddatum kan worden opgegeven. Wanneer je geen einddatum wilt opgeven, is de request body een leeg object: 
+```
+{ }
+```
+
+Je kan een volgindicatie beëindigen door de einddatum voor het volgen op te nemen in de request body. Bijvoorbeeld: 
+```
+{
+  "einddatum": "2022-04-19"
+}
+```
+
+Op elke werkdag worden er wijzigingen ontvangen voor enkele burgerservicenummers. Je kunt de volgende burgerservicenummers gebruiken:
+| Weekdag   | burgerservicenummers |
+|---------- |--------------------- |
+| maandag   | 999994669, 999992740 |
+| dinsdag   | 999990019, 999990925 |
+| woensdag  | 999993276, 999993252 |
+| donderdag | 999991176, 999993136 |
+| vrijdag   | 999993215, 999990317 |
+| zaterdag  | 999994281, 999990743 |
+| zondag    | 999993070, 999991334 |
+
+Vervolgens kun je wijzigingen opvragen met GET /wijzigingen. Daarbij kan je de query-parameter 'vanaf' gebruiken om alleen wijzigingen te ontvangen vanaf de opgegeven datum.
+
+Je zult pas wijzigingen ontvangen van na het moment dat je de volgindicatie hebt gezet. Wanneer je vandaag een volgindicatie toevoegt op een burgerservicenummer, dan zal je dus nog niet direct wijzigingen ontvangen voor die persoon.
```

Comment:
De 'van' is waarschijnlijk teveel in 'Je zult pas wijzigingen ontvangen *van* na het moment'